rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/LSPIDEServices.java

```diff
@@ -92,7 +92,7 @@ public void browse(URI uri, String title, int viewColumn) {
     public void edit(ISourceLocation path) {
         ISourceLocation physical = Locations.toClientLocation(path);
         ShowDocumentParams params = new ShowDocumentParams(physical.getURI().toASCIIString());
-        params.setTakeFocus(true);
+        params.setTakeFocus(false);
```

Comment:
I don't think we should switch this off, since this changes the behavior of the editor that would come forward.

Reply:
This shouldn't be here indeed. Bad commit